src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -92,6 +92,7 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
   @Prop({ required: true }) readonly index!: number
 
   @Getter(useBusinessStore) isBenBcCccUlc!: boolean
+  // @Getter(useAuthenticationStore) isRoleStaff!: boolean
```

Comment:
Used in this component but inherited from a mixin. Still, I like to see the declaration in the component.